### PR TITLE
Tell poetry to install receptor from the devel branch

### PR DESCRIPTION
**Pull request by squidboylan**
It seems like poetry was defaulting to the master branch for some
reason, which is no longer the default branch, this was causing
dependency resolution problems when trying to run tests on receptor
devel branch

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -23,13 +23,13 @@ click = "^7.0"
...
 python = "^3.6"
 pyyaml = "^5.3"
-receptor = { git = "https://github.com/project-receptor/receptor.git" }
 requests = "^2.22.0"
 setuptools = "*"  # For pkg_resources
 wait-for = "^1.1.1"
 python-dateutil = "^2.8.1"
 prometheus-client = "^0.7.1"
 psutil = "^5.6.7"
+receptor = {git = "https://github.com/project-receptor/receptor.git", rev = "devel"}
```
Comment (jeremy-audet) [inline pyproject.toml]: Shouldn't this use the "branch" keyword, not the "rev" keyword? See: https://python-poetry.org/docs/dependency-specification/#git-dependencies

**Review by jeremy-audet — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (squidboylan) [reply pyproject.toml]: yep! updated

Comment (squidboylan) [reply pyproject.toml]: updated and also reran `poetry lock` again just in case

**Review by jeremy-audet — APPROVED** [review]
Looks reasonable to me. I'm unable to install this application with `poetry install`, but that also holds true for the master branch, so I'm not going to block on that issue.

**Pull request merged**